bionic/aip/main.py

```diff
@@ -0,0 +1,70 @@
+"""
+This module is run as main in order to execute a task on a worker.
+"""
+
+import logging
+import os
+import sys
+
+from bionic.deps.optdep import import_optional_dependency
+
+
+def run():
+    # Scope the import to this function to avoid raising for anyone not using it.
+    blocks = import_optional_dependency("blocks", raise_on_missing=True)
+    cloudpickle = import_optional_dependency("cloudpickle", raise_on_missing=True)
+
+    ipath = sys.argv[-1]
+    fs = blocks.filesystem.GCSNativeFileSystem()
+    with fs.open(ipath, "rb") as f:
+        task = cloudpickle.load(f)
+
+    # Now that we have the task, setup logging.
```

Comment:
nit: I know this is from Conductor, but "setup" is a noun. The verb is "set up".